stores/migrations.go

```diff
@@ -339,3 +347,38 @@ func performMigration00003_healthcache(txn *gorm.DB, logger glogger.Interface) e
 	logger.Info(context.Background(), "migration 00003_healthcheck complete")
 	return nil
 }
+
+func performMigration00004_uploadPacking(txn *gorm.DB, logger glogger.Interface) error {
+	m := txn.Migrator()
+
+	// Disable foreign keys in SQLite to avoid issues with updating constraints.
+	if isSQLite(txn) {
+		if err := txn.Exec(`PRAGMA foreign_keys = 0`).Error; err != nil {
+			return err
+		}
+	}
+
+	if m.HasColumn(&dbBufferedSlab{}, "db_slab_id") {
+		// Drop buffered slabs since the schema has changed and the table was
```

Comment:
Comment doesn't match the code here? You're not dropping the `buffered_slabs` table. Not that big of an issue but since it's migration code it's best to update the comment here I think.